pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -51,6 +52,8 @@
 from sktime.utils.validation.forecasting import check_y_X  # type: ignore
 from sktime.forecasting.model_selection import SlidingWindowSplitter  # type: ignore
 
+from ..tests.time_series import test_
```

Comment:
Can we do an absolute import here for consistency?

Reply:
done